### DISCUSSION TITLE
SD Card Adapter

### DISCUSSION THREAD

**Discussion by Cal-LF** [Jyers/Marlin#92] [Ideas]
Many E3V2 owners use an SD card adapter. The tiny micro cards are a pain to use. When using the adapter, the card has to be installed when powering up to read the gcode file. The printer needs to be turned off and back on again when changing the SD card in order to read the file.
I found that changing one line in Configuration.h will allow the card to be read without powering down the printer. It will read the SD card by scrolling the main menu highlight to the right and back to PRINT again. Could you verify this and add this to all your compiled files so SD card adapter users can have this improvement?
![sd](https://user-images.githubusercontent.com/76499230/108633830-74234a00-7444-11eb-8d7a-68d03c214e7e.jpg)

**Comment by Jyers** [comment]
The SD_CHECK_AND_RETRY option is purely for dealing with errors in SD communications, as in during printing. This has nothing to do with an SD card being recognized. I programmed the display to remount the SD when the print menu is drawn for the very purpose of fixing this issue for those using sd adapters. The check and retry setting is not required for the fix.

**Reply by Cal-LF** [reply]
Thank you for responding. So noted on the SD card fix. I thought this change fixed it. I did not see this anywhere in the updates notes.

**Reply by Jyers** [reply]
It was added a while ago, before I had proper release notes
